client/src/common/components/Comments/Comment.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { CohortIcon } from 'assets/images/icons';
+
+const Comment = ({ author, comment, avatar }) => (
+  <div className="comment">
+    <div className="comment__avatar">
+      {/* Cohort icon is temporary for now, it will be user avatar here */}
+      <CohortIcon />
```

Comment:
maybe just use `{ avatar ? avatar : <PersonIcon /> }`- check person icon in material ui and remove comment?